.github/workflows/aws-lambda-java-serialization.yml

```diff
@@ -26,6 +26,9 @@ jobs:
         java-version: 1.8
     
     # Install base module
+    - name: Build events with Maven
+      run: mvn -B install --file aws-lambda-java-events/pom.xml
+    # Install target module
     - name: Build serialization with Maven
       run: mvn -B install --file aws-lambda-java-serialization/pom.xml
```

Comment:
nit: Why install this target module when others are packaged instead?

Reply:
I did this because the next step is to test `aws-lambda-java-runtime-interface-client`, which is a consumer of this package and I thought it would be good for `aws-lambda-java-serialization` to  be installed in the local repository.

Reply:
Right, gotcha 👍